lib/src/comment_references/parser.dart

```diff
@@ -65,7 +66,7 @@ class StringTrie {
   }
 }
 
-late final StringTrie operatorParseTrie = () {
```

Comment:
This changes the semantics, yes? It is no longer lazily evaluated, e.g. in

```
class C {
  final f = () {
    print('yay');
    return 7;
  }();
}

void main() {
  C();
  print('boo');
  C().f;
}
```

we get "yay" "boo" "yay" (rather than "yay" "boo"). Is this trie used unconditionally?

Reply:
Since this is a top-level field with an initializer it is already evaluated lazily. I added [`unnecessary_late`](https://dart-lang.github.io/linter/lints/unnecessary_late.html) to catch this but that won't be included until 2.16.

From the spec:
> Static variable declarations with an initializing expression are initialized
lazily (8.1).

Reply:
Ah I see. It looked indented by two spaces but I think its just the diff `- ` text.